### DISCUSSION TITLE
Has the Webp vulnerability been addressed yet?

### DISCUSSION THREAD

**Discussion by Telmesomething** [Floorp-Projects/Floorp#474] [Feature Request]
According to the [Ghacks.net](https://www.ghacks.net/2023/09/13/mozilla-patches-critical-webp-security-issue-in-firefox-and-thunderbird/) site both Mozilla Firefox and Google Chrome have addressed a serious vulnerability in Webp, but I didn't see any mention of it in the latest Floorp release notes. 

What's the position now?

**Comment by creeper-0910** [comment]
Basically Floorp is based on Firefox ESR 115 update.
If it is fixed in ESR 115, it is basically fixed.

**Comment by Ryosuke-Asano** [comment]
> According to the [Ghacks.net](https://www.ghacks.net/2023/09/13/mozilla-patches-critical-webp-security-issue-in-firefox-and-thunderbird/) site both Mozilla Firefox and Google Chrome have addressed a serious vulnerability in Webp, but I didn't see any mention of it in the latest Floorp release notes.
> 
> What's the position now?

We are so fast. The security fix applied at version 11.3.3.
https://github.com/Floorp-Projects/Floorp/releases/tag/v11.3.3

**Comment by Telmesomething** [comment]
My understanding of it is that it wasn't fixed until the 115.2.1 release. 
https://www.mozilla.org/en-US/security/advisories/mfsa2023-40/

But maybe you meant that version. 

(Sorry, but I couldn't find the link on Github hence this reply using email). 


-- 
 Sent with Tutanota; enjoy secure & ad-free emails: 
 https://tutanota.com


Sep 26, 2023, 16:36 by ***@***.***:

**Reply by creeper-0910** [reply]
The version of Floorp is different from Firefox.

**Comment by Telmesomething** [comment]
Yes, I've noticed that. Using Floorp I can read news media without the constant prompts to "signup to continue reading" that frequently appear on The Guardian <https://www.theguardian.com/international> newspaper site when using Firefox. 

Anyway, I decided to install the latest version (11.4.1) which, to my surprise works much better than 10.16.0 even though I'm still using Windows 8.1

Thanks once again for responding. 

Terence Verney
-- 
 Sent with Tutanota; enjoy secure & ad-free emails: 
 https://tutanota.com


Sep 29, 2023, 09:34 by ***@***.***:
